ui/raidboss/data/06-ew/raid/p2s.ts

```diff
@@ -0,0 +1,371 @@
+          return output.w!();
+        if (xCoord > 100)
+          return output.e!();
+      },
+      outputStrings: {
+        e: Outputs.east,
+        w: Outputs.west,
+      },
+    },
+    {
+      // Spread aoe marker on some players, not all
+      id: 'P2S Tainted Flood',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6838', source: 'Hippokampos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6838', source: 'Hippokampos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6838', source: 'Hippokampos' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6838', source: 'ヒッポカムポス' }),
+      condition: (data, matches) => matches.target === data.me,
+      response: Responses.spread(),
+    },
+    {
+      id: 'P2S Coherence Flare',
+      type: 'Tether',
+      // Whoever has tether when cast of 681B ends will be flared
+      netRegex: NetRegexes.tether({ id: '0054', source: 'Hippokampos' }),
+      netRegexDe: NetRegexes.tether({ id: '0054', source: 'Hippokampos' }),
+      netRegexFr: NetRegexes.tether({ id: '0054', source: 'Hippokampos' }),
+      netRegexJa: NetRegexes.tether({ id: '0054', source: 'ヒッポカムポス' }),
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.text!(),
+      run: (data, matches) => data.flareTarget = matches.target,
+      outputStrings: {
+        text: {
+          en: 'Flare Tether',
+        },
+      },
+    },
+    {
+      id: 'P2S Coherence Stack',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '681B', source: 'Hippokampos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '681B', source: 'Hippokampos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '681B', source: 'Hippokampos' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '681B', source: 'ヒッポカムポス' }),
+      condition: (data) => data.flareTarget !== data.me,
+      // 12 second cast, delay for tether to settle
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 6,
+      alertText: (data, _matches, output) => {
+        if (data.role === 'tank')
+          return output.flareLineTank!();
+        return output.flareLineStack!();
+      },
+      outputStrings: {
+        flareLineStack: {
+          en: 'Line Stack (behind tank)',
+        },
+        flareLineTank: {
+          en: 'Line Stack (be in front)',
+        },
+      },
+    },
+    {
+      // Raidwide knockback -> dont get knocked into slurry
+      id: 'P2S Shockwave',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '682F', source: 'Hippokampos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '682F', source: 'Hippokampos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '682F', source: 'Hippokampos' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '682F', source: 'ヒッポカムポス' }),
+      // 7.7 cast time, delay for proper arm's length
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 2.7,

```

Comment:
I think this math is wrong.  If the cast time is 7.7 (really 8), then you need a delay of 2.7.  Your math would give only 2.7s of reaction time, instead of 5s.

```suggestion
      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
```

Reply:
Sorry, realized this right before the suggestion.